scripts/check-bins.py

```diff
@@ -36,6 +36,12 @@ def check_jemalloc(executable):
         pr("error: %s does not contain jemalloc\n" % executable)
         sys.exit(1)
 
+def is_sse_enabled(features):
+    return not features or "sse" in features
```

Comment:
Actually `sse` is not enabled until declared explicitly.

Reply:
Sorry, I don't get the point what this refer to.  Here https://github.com/tikv/tikv/blob/74854810ac3de2cd60c0566384e09d63c5b1b2d2/Makefile#L61-L63 in the Makefile, if we don't set `ROCKSDB_SYS_SSE =0` explicitly, feature `sse` will be enabled.

Reply:
It's enabled because it's added to `ENABLE_FEATURES`, in other word, declared explicitly. `not features` is true means no features is declared, then sse should be disabled.

Reply:
OK. I thought if `features` is empty, it means this check is run manually, so we should do check by default. Seemly this does not make much sense, I removed the `not features` logic for all the enabled check